src/renderer/prefComponents/image/index.vue

```diff
@@ -84,8 +90,8 @@ export default {
     openImageFolder () {
       shell.openPath(this.imageFolderPath)
     },
-    modifyImageFolderPath () {
-      return this.$store.dispatch('SET_IMAGE_FOLDER_PATH')
+    modifyImageFolderPath (value) {
+      return this.$store.dispatch('SET_IMAGE_FOLDER_PATH', value)
```

Comment:
This event commits the path to the main-process and create the directory by default. We have to remove the directory creation in `src/main` and verify that the renderer process creates it if not already exists.